daemons/pacemakerd/pacemakerd.c

```diff
@@ -425,10 +425,7 @@ main(int argc, char **argv)
 #endif
 
     if (pcmk__locate_sbd() > 0) {
-        setenv("PCMK_watchdog", "true", 1);
         running_with_sbd = TRUE;
-    } else {
-        setenv("PCMK_watchdog", "false", 1);
```

Comment:
Interestingly, sbd/src/sbd-pacemaker.c also sets it, but sbd doesn't use it either

Reply:
It's left over from an older implementation (see f557164 when it was dropped). Pacemaker didn't drop it when the implementation changed, so I'm not surprised that sbd didn't either :)